packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -182,10 +182,10 @@ const ProductList = (props) => {
 
     /**************** Einstein ****************/
     useEffect(() => {
-        if (searchQuery) {
-            einstein.sendViewSearch(searchQuery, productSearchResult)
-        } else {
-            einstein.sendViewCategory(category, productSearchResult)
+        if (productSearchResult) {
```

Comment:
should it still guard `searchQuery`?

like 

```suggestion
        if (productSearchResult && searchQuery) {
```

Reply:
oh never mind, i see the ternary below handles it